app/main.py

```diff
@@ -1,2 +1,67 @@
+from __future__ import annotations
+from typing import Hashable, Any
+from dataclasses import dataclass
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: Any
+    hash_el: int
+
+
 class Dictionary:
-    pass
+    INITIAL_CAPACITY = 8
+    THRESHOLD = 2 / 3
```

Comment:
The `THRESHOLD` is calculated using integer division, which will result in zero. This will cause the resizing logic to never trigger. Consider using a float division instead: `THRESHOLD = 2 / 3`.